src/cli/index.js

```diff
@@ -50,6 +51,14 @@ export default class Cli {
   async run() {
     await validateInstall(this.cwd)
     const configuration = await this.getConfiguration()
+    if (configuration.listI18nLanguages) {
```

Comment:
what about passing `stdout` to the `print*` methods and let them do the actual `write`s?